cypress/support/index.js

```diff
@@ -1,3 +1,11 @@
+// FIXME a better way to wait instead of time?
+/* Now that we are using the jsonld.js lib, network calls for compaction/expansion take some time to complete
+ * So we need to wait before testing. */
```

Comment:
We shouldn't do network calls from tests, it would cause the CI to randomly break, and it can be heavy on the server we query.

All contexts needed by tests should be available locally (through a document loader) to reduce issues for users, too